packages/perseus-core/src/analytics.ts

```diff
@@ -18,13 +18,28 @@ export type PerseusAnalyticsEvent =
           type: "perseus:expression-focused";
           payload: null;
       }
+    // TODO(catjohnson): Remove this error type in LEMS-2826
```

Comment:
Nit: I think you can rewrite this comment as `TODO(LEMS-2826): Remove this error type` just in case anything changes and someone else ends up doing this part.

And then you can also comment in your ticket "Look for `TODO(LEMS-2826)`"

\+ Same for all the comments below, which I think might be missing the ticket number